src/components/showcase/ShowcaseHeader.js

```diff
@@ -153,7 +153,9 @@ export function Header() {
   const { handleSetSeedPhrase, handlePressImportButton } = useImportingWallet();
 
   const onWatchAddress = useCallback(() => {
-    contextValue?.setIsSearchModeEnabled(false);
+    if (contextValue?.setIsSearchModeEnabled) {
+      contextValue?.setIsSearchModeEnabled(false);
```

Comment:
You don't need the optional here since you're already checking on the line above